core/src/main/java/org/apache/calcite/rel/externalize/RelJson.java

```diff
@@ -995,7 +995,7 @@ private List<RexNode> toRexList(RelInput relInput, List operands) {
     operatorTable.lookupOperatorOverloads(
         new SqlIdentifier(name, new SqlParserPos(0, 0)),
         null,
-        sqlSyntax,
+        sqlSyntax.family,
```

Comment:
Why use `family` now? 

Could this change the expected output of JSON -> Rex translations?

Reply:
This change came from one of Julian's commits. I think it's to account for the discrepancy from his last comment on CALCITE-6021. pushing a commit momentarily to try and fix this

Reply:
See https://github.com/apache/calcite/pull/3457/commits/31fb1970ab558eabc308f7d4ab18f18dace36937 for my fix for this

Reply:
I agree with @tjbanghart that this change is mysterious. You should explain it in a commit comment.

Something like the fact that CURRENT_DATETIME has FUNCTION_ID syntax (to allow being called without parentheses) but is registered in the op table as a FUNCTION. I don't recall, but it's worth digging up the rationale.

Reply:
I removed this change in a later commit